.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "rust-analyzer.cargo.features": "all"
+}
```

Comment:
Was this part of the diff intentional? I think we'd prefer to see a `.vscode` addition to `.gitignore` in a stand-alone commit and this file backed out. There isn't an established precedent for committing IDE settings to the Rustls repos AFAIK.

Reply:
Unintentional, but it might take me a bit to get to it now I've got some life stuff occuring

Reply:
No worries, best of luck.